jmclient/jmclient/wallet.py

```diff
@@ -674,10 +675,17 @@ def select_utxos(self, mixdepth, amount, utxo_filter=None,
             amount: int, total minimum amount of all selected utxos
             utxo_filter: list of (txid, index), utxos not to select
             maxheight: only select utxos with blockheight <= this.
+            require_auth_address: if True, output utxos must include a
+                wallet-native address. The first item of the output dict is
```

Comment:
may as well change from `wallet-native` to `standard wallet` here for consistency, I guess.